sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RMResources.java

```diff
@@ -29,18 +29,18 @@
     public static final String ResourceTokenNotFound = "Resource token not found.";
     public static final String Unauthorized = "Unable to authenticate the request. The request requires valid user authentication.";
     public static final String Forbidden = "Unable to proceed with the request. Please check the authorization claims to ensure the required permissions to process the request.";
-    public static final String NotFound = "Entity with the specified id does not exist in the system.";
+    public static final String NotFound = "Entity with the specified id does not exist in the system. More info: https://aka.ms/cosmosdb-tsg-not-found";
```

Comment:
This should already be included in the backend error message. In what scenario would the SDK be generating this message?

Reply:
There are some instances in `AddressResolver` code peice which throw NotFound exceptions with this message. 
Also reading the TSG indicates that this can be thrown by SDK code : https://docs.microsoft.com/en-us/azure/cosmos-db/troubleshoot-not-found